controllers/email-l10n.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+const HBSHelpers = require("../hbs-helpers");
+
+
+function testEmails(req, res) {
+  const unsafeBreachesForEmail = [];
+  const email = "example@email.com";
+  let breachAlert;
+  let buttonValue = req.fluentFormat("verify-my-email");
+
+  if (!req.query.view) {
+    req.query.view = "email_verify";
+  }
+
+  if (req.query.type) {
+    buttonValue = req.fluentFormat("report-scan-another-email");
+    if(req.query.type === "singleBreach" || req.query.type === "breachAlert") {
+      unsafeBreachesForEmail.push(req.app.locals.breaches.filter(breach => breach.Name === "Experian")[0]);
+      if(req.query.type === "breachAlert") {
+        breachAlert = unsafeBreachesForEmail[0];
+      }
+    } else if (req.query.type === "multipleBreaches") {
+      const breachArray = ["Experian", "Dropbox", "Apollo"];
```

Comment:
What is this hardcoded list of breaches for?
I see we're looping over it below, but curious why we need to hardcoded 3 breach names here.

Reply:
We really only actually need two but it's there so that translators can see an example of a report with multiple breaches as well as a singular breach since the strings are different.